### PR TITLE
added new units to the TemperatureChangeRate quantity

### PR DESCRIPTION
**Pull request by David-HH-Dev**
added the following units to TemperatureChangeRate:
DegreeFahrenheitPerMinute
DegreeFahrenheitPerSecond
DegreeKelvinPerSecond
DegreeCelsiusPerHour
DegreeKelvinPerHour
DegreeFahrenheitPerHour

After reviewing the unit tests I came to the conclusion that the recently added (https://github.com/angularsen/UnitsNet/pull/1429) unit DegreeKelvinPerMinute had a wrong conversion.

I checked that for example here:
https://www.unitsconverters.com/en/Rate-Of-Temperature-Change-Conversions/Measurement-1189
and also here:
https://www.quora.com/Why-is-32-not-used-when-converting-per-degree-Celsius-C-1-to-per-degree-Fahrenheit-F-1

### CODE REVIEW

File: Common/UnitDefinitions/TemperatureChangeRate.json [hunk 1]
```diff
@@ -35,15 +35,89 @@
     {
       "SingularName": "DegreeKelvinPerMinute",
       "PluralName": "DegreesKelvinPerMinute",
-      "FromUnitToBaseFunc": "({x} / 60) - 273.15",
-      "FromBaseToUnitFunc": "({x} + 273.15) * 60",
+      "FromUnitToBaseFunc": "{x} / 60",
+      "FromBaseToUnitFunc": "{x} * 60",
```
Comment (angularsen) [inline Common/UnitDefinitions/TemperatureChangeRate.json:39]: Good catch, I missed this when I last reviewed this by checking google, which gives the wrong answer:
https://www.google.com/search?q=DegreesKelvinPerMinute+In+1+DegreeCelsiusPerSecond&rlz=1C1ONGR_enNO1008NO1008&oq=DegreesKelvinPerMinute+In+1+DegreeCelsiusPerSecond&gs_lcrp=EgZjaHJvbWUqBggAEEUYOzIGCAAQRRg70gEINTE2NmowajeoAgCwAgA&sourceid=chrome&ie=UTF-8

It totally makes sense for kelvin and celsius rate of change to be equal.

**Review by angularsen — APPROVED** [review]

File: Common/UnitDefinitions/TemperatureChangeRate.json (hunk 1 above)
Comment (David-HH-Dev) [reply Common/UnitDefinitions/TemperatureChangeRate.json:39]: @angularsen thanks for merging my PR. Do you know (roughly) when you will release a new Nuget package?

**Comment by angularsen** [comment]
Thanks! Nuget should be out shortly.

[Release UnitsNet/5.61.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F5.61.0)

File: Common/UnitDefinitions/TemperatureChangeRate.json (hunk 1 above)
Comment (angularsen) [reply Common/UnitDefinitions/TemperatureChangeRate.json:39]: Thanks for the reminder, I got busy and forgot it (again). Maybe it is my age 🤔

**Comment by angularsen** [comment]
@David-HH-Dev Thanks for checking in, turns out there was a build error due to some new vulnerability in a Nuget.Protocol nuget package (meta!)

I just pushed a fix, let's see if the nuget gets deployed in the next 30 minutes or so.
https://dev.azure.com/unitsnet/Units.NET/_build/results?buildId=932&view=results

**Comment by angularsen** [comment]
Another brand new compile error 🙄 
Second attempt
https://dev.azure.com/unitsnet/Units.NET/_build/results?buildId=936&view=results

File: Common/UnitDefinitions/TemperatureChangeRate.json (hunk 1 above)
Comment (andrewmeservy) [reply Common/UnitDefinitions/TemperatureChangeRate.json:39]: ahhh thank you for catching my mistake, as well as the links explaining the correct calculation!!

**Comment by angularsen** [comment]
Merged into V6 branch, nuget should be out shortly.

[Release UnitsNet/6.0.0-pre013 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F6.0.0-pre013)

**Pull request merged**